uefi/flash.go

```diff
@@ -141,6 +141,21 @@ type FlashImage struct {
 	regions     []Firmware
 }
 
+func (n *FlashImage) Apply(v Visitor) error {
+	return v.VisitImage(n)
+}
+
+func (n *FlashImage) ApplyChildren(v Visitor) error {
+	if n.BIOS != nil {
```

Comment:
Why are we only applying this to BIOSRegion? Shouldn't we visit all children? IFD, BIOS, ME, GBE, PD?

Reply:
Done! Although, the visitors a bit verbose now. I might make a separate Visitor interface for just the intel nodes.